webservices/common/models/efiling.py

```diff
@@ -0,0 +1,118 @@
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import ARRAY, TSVECTOR
+from sqlalchemy.ext.declarative import declared_attr
+
+from webservices import docs
+from .base import db, BaseModel
+
+class BaseFilingSummary(db.Model):
+    __tablename__ = 'real_efile_summary'
+    file_number = db.Column('repid', db.Integer, index=True, primary_key=True)
+    line_number = db.Column('lineno', db.Integer, primary_key=True)
+    column_a = db.Column('cola', db.Float)
+    column_b = db.Column('colb', db.Float)
+
+class BaseFiling(db.Model):
+    __abstract__ = True
+    file_number = db.Column('repid', db.Integer, index=True, primary_key=True)
+    committee_id = db.Column('comid', db.String, index=True, doc=docs.COMMITTEE_ID)
+    coverage_start_date = db.Column('from_date', db.Date)
+    coverage_end_date = db.Column('through_date', db.Date)
+    rpt_pgi = db.Column('rptpgi', db.String, doc=docs.ELECTION_TYPE)
+    report_type = db.Column('rptcode', db.String)
+    image_number = db.Column('imageno', db.Integer)
+    street_address_one = db.Column('str1', db.String)
+    street_address_two = db.Column('str2', db.String)
+    city = db.Column(db.String)
+    state = db.Column(db.String)
+    zip = db.Column(db.String)
+    election_date = db.Column('el_date', db.Date)
+    election_state = db.Column('el_state', db.String)
+    create_date = db.Column('create_dt', db.Date)
+    sign_date = db.Column(db.Date)
+
+
+class BaseF3PFiling(BaseFiling):
+    __tablename__ = 'real_efile_f3p'
+    file_number = db.Column('repid', db.Integer, index=True, primary_key=True)
+
+    treasurer_last_name = db.Column('lname', db.String)
+    treasurer_middle_name = db.Column('mname', db.String)
+    treasurer_first_name = db.Column('fname', db.String)
+    prefix = db.Column(db.String)
+    suffix = db.Column(db.String)
+    committee_name = db.Column('c_name', db.String, index=True, doc=docs.CANDIDATE_NAME)
+    street_address_one = db.Column('c_str1', db.String)
+    street_address_two = db.Column('c_str2', db.String)
+    city = db.Column('c_city', db.String)
+    state = db.Column('c_state', db.String)
+    zip = db.Column('c_zip', db.String)
+    total_receipts = db.Column('tot_rec', db.Float)
```

Comment:
Yeah, and I think I made a comment about that somewhere.  But the touchy thing about is the summary tables that _some_ of these models map to have a total_receipts_ytd and total_receipts_per, and same for disbursements.  So, since they are in fact all different values, I don't believe that they should be called ytd or per, or if they are then what distinguishes them from the columns from col_a and col_b?